src/drivers/ambapp/ambapp_id.h

```diff
@@ -10,8 +10,6 @@
 #ifndef AMBAPP_H_
 #define AMBAPP_H_
 
-#include <module/embox/driver/ambapp_api.h>
```

Comment:
Why is this required?
If it rm it can break sparc templates

Reply:
fixed